src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class Lottery {
+    private final int randomNumber = new SecureRandom().nextInt(100);
```

Comment:
The randomNumber of instances of this class is hardcoded, so it's impossible to reuse them. It's better to get a random number on each getRandomBall() call.